server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionFinalizer.java

```diff
@@ -128,7 +128,8 @@ private void notifyTserver(Location loc, ExternalCompactionFinalState ecfs) {
             ecfs.getExternalCompactionId().canonical(), ecfs.getExtent().toThrift(),
             ecfs.getFileSize(), ecfs.getEntries());
       } else if (ecfs.getFinalState() == FinalState.FAILED) {
-        LOG.debug("Notifying tserver {} that compaction {} has failed.", loc, ecfs);
+        LOG.debug("Notifying tserver {} that compaction {} with {} has failed.", loc, ecfs,
+            ecfs.getExternalCompactionId());
```

Comment:
move ID to right after `compaction` in log message

```suggestion
        LOG.debug("Notifying tserver {} that compaction {} with {} has failed.", loc, cfs.getExternalCompactionId(), 
         ecfs);
```